packages/web-components/fast-foundation/src/menu/menu.ts

```diff
@@ -129,6 +129,9 @@ export class Menu extends FASTElement {
 
     private setItems = (): void => {
         const focusIndex = this.menuItems.findIndex(this.isFocusableElement);
+        const hasRole =
```

Comment:
I'd prefer a different const name here - technically all of these could have explicit roles, including a valid role of simply `menuitem`.

Reply:
this is removed